designer/server/src/routes/forms/api.js

```diff
@@ -33,16 +33,17 @@ export default [
         parse: true
       },
       async handler(request, h) {
-        const { id } = request.params
+        const { auth, params, payload } = request
+        const { id } = params
+        const author = forms.getAuthor(auth.credentials)
+
         try {
-          const { value, error } = formDefinitionSchema.validate(
-            request.payload,
-            {
-              abortEarly: false
-            }
-          )
+          const result = formDefinitionSchema.validate(payload, {
+            abortEarly: false
+          })
 
-          if (error) {
+          if (result.error) {
+            const error = result.error
```

Comment:
Similar with this, feels like it didn't need another const?

```suggestion
```

<br>

Using `result.error` is a bit more obvious

```patch
-             [error, request.payload]
+             [result.error, request.payload]
```

Reply:
I have a rule, if I reference an object property more than once, I'll create a variable for it.
I find that it keeps things more readable! Personally, I don't like `params.this` and `request.that` :see_no_evil: 

Also, there's a cost to the JS interpreter each time you look up an key on an object. While these days it has probably been optimised out in the JIT compiler, old habits and all.